src/util/grunt_types.json

```diff
@@ -1,291 +1,258 @@
 {
   "1": {
     "type": "",
-    "grunt": "Blanche",
-    "gender": 2
+    "grunt": "Blanche"
   },
   "2": {
     "type": "",
-    "grunt": "Candela",
-    "gender": 2
+    "grunt": "Candela"
   },
   "3": {
     "type": "",
-    "grunt": "Spark",
-    "gender": 1
+    "grunt": "Spark"
   },
   "4": {
-    "type": "Mixed",
+    "type": "",
     "grunt": "Male Grunt",
-    "gender": 1,
-    "second_reward": true,
+    "second_reward": "true",
     "encounters":{
-        "first": ["1","4","7"],
-        "second":["2","5","8"],
-        "third":["3","6","9"]
+        "first": ["001_00","004_00","007_00"],
+        "second":["002_00","005_00","008_00"],
+        "third":["003_00","006_00","009_00"]
     }
   },
   "5": {
-    "type": "Mixed",
```

Comment:
Poracle needs to call these mixed to  be able to track them

Reply:
oops